lb.tf

```diff
@@ -20,7 +21,7 @@ resource "cloudscale_server" "lb" {
   zone_slug        = "${var.region}1"
   flavor_slug      = "plus-8"
   image_slug       = "ubuntu-20.04"
-  server_group_ids = [cloudscale_server_group.lb.id]
+  server_group_ids = var.lb_count != 0 ? [cloudscale_server_group.lb[0].id] : []
```

Comment:
Is this really necessary? Will this even be evaluated if the `count` for this resource is `0`?

Reply:
If the count is zero, there will be no zero index for that resource and thus it will fail.

Reply:
Yes, I just thought since the count is 0 it wouldn't even try to create the resource so it doesn't matter that it's accessing an nonexistent index.